package.json

```diff
@@ -192,7 +192,7 @@
     "react-markdown": "^3.4.1",
     "react-mfb": "^0.6.0",
     "react-motion": "^0.4.7",
-    "react-mutation-mapper": "^0.2.6",
+    "react-mutation-mapper": "^0.3.0-beta.13",
```

Comment:
This should be 0.3.0?

Reply:
yes, I will release 0.3.0 and update this together with the screenshot tests, once everyone is happy with the changes.